code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -51,9 +51,9 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		if(reac_volume >= 5)
 			var/obj/item/book/affectedbook = O
 			affectedbook.dat = null
-			to_chat(usr, "<span class='notice'>Through thorough application, you wash away [affectedbook]'s writing.</span>")
+			O.visible_message("<span class='notice'>\The [O]'s writing is washed away by [id]!</span>")
```

Comment:
If you use [id] you'll get stuff like "somereagent" or "some_reagent", just use the name. Also the \The at the beginning shouldn't be necessary, it's added automatically.